### PR TITLE
Implement createBuilding game command

### PR DESCRIPTION
**Pull request by duncanspumpkin**
should work. Not fully tested.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Seems to work on replays

File: src/OpenLoco/src/GameCommands/Buildings/CreateBuilding.cpp [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+#include "S5/S5.h"
+#include "ViewportManager.h"
+#include "World/Industry.h"
+#include "World/Station.h"
+#include "World/TownManager.h"
+
+namespace OpenLoco::GameCommands
+{
+    // 0x0042D133
+    static uint32_t createBuilding(const BuildingPlacementArgs& args, uint8_t flags)
```
Comment (AaronVanGeffen) [inline src/OpenLoco/src/GameCommands/Buildings/CreateBuilding.cpp:28]: Does this function require hooking? It seems to be called directly from aiThink. Might be used directly from one of the town growth functions as well.

**Review by AaronVanGeffen — CHANGES_REQUESTED** [review]
Building ghosts destroy existing buildings. Hilarious, but not quite correct :P

**Review by AaronVanGeffen — APPROVED** [review]
All good now, as far as I'm concerned.

File: src/OpenLoco/src/GameCommands/Buildings/CreateBuilding.cpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/OpenLoco/src/GameCommands/Buildings/CreateBuilding.cpp:28]: No we have implemented all callers. For once it seems things actually go via doCommand.

**Pull request merged**